js-foundations.js

```diff
@@ -1,50 +1,151 @@
 // ASSESSMENT 2: JAVASCRIPT FOUNDATIONS
 // Coding practical questions
 
-// 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
-
-
-
-// 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
-
-
+// // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
+const divisibleBy = (num) => {
+    if(num%3 === 0){
+        return `${num} is divisble by 3`
+    } else {
+        return `${num} is not divisibleBy`
+    }
+}
+console.log(divisibleBy(15))
+console.log(divisibleBy(0))
+console.log(divisibleBy(385))
+console.log(divisibleBy(-7))
+console.log(divisibleBy(-9))
+
+// // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
+const helloMe = {
+    firstName: 'Christi', 
+    lastName: 'Wright', 
+    origin: 'Bay Area', 
+    alum: 'UCSD',
+    getData: function () {return `${helloMe.firstName} ${helloMe.lastName} is from the ${helloMe.origin} and went to college at ${helloMe.alum}.`}
+}
+
+console.log(helloMe.getData())
 
 // ------------------------------------- Consider the variable:
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 
-// 3a. Write a function that returns every other item from the array as one string. Expected output: "streetlamp teeth nephew database"
+// // 3a. Write a function that returns every other item from the array as one string. Expected output: "streetlamp teeth nephew database"
+ 
+//filter - need to update to function, currently returning the whole array
+// const arrayToString = (array) => {
+//     return array.filter(word => word ).join(" ") 
+//  }
+// console.log(arrayToString(randomNouns))
+
```

Comment:
I'm glad you did this two different ways! On the filter one, you just need to add an evaluation on the word => word part. You did it correctly in the evaluation of the for loop (i%2 === 0)